### PR TITLE
fixing Weathermap config permissions

### PR DESCRIPTION
**Pull request by LoveSkylark**
Weathermap is unable to run because of file ownership , this rearranges the RUN order a little and add a chown for the weathermap folder.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -118,8 +118,9 @@ RUN apk --update --no-cache add -t build-dependencies \
   && sed -i '/runningUser/d' lnms \
   && echo "foreach (glob(\"/data/config/*.php\") as \$filename) include \$filename;" >> config.php \
   && echo "foreach (glob(\"${LIBRENMS_PATH}/config.d/*.php\") as \$filename) include \$filename;" >> config.php \
-  && git clone https://github.com/librenms-plugins/Weathermap.git ./html/plugins/Weathermap \
   && chown -R nobody.nogroup ${LIBRENMS_PATH} \
+  && git clone https://github.com/librenms-plugins/Weathermap.git ./html/plugins/Weathermap \
+  && chown -R librenms:librenms ./html/plugins/Weathermap \
```
Comment (murrant) [inline Dockerfile:123]: I suggest clone it where it is, then change the owner to librenms:librenms for these required directories only:
./html/plugins/Weathermap/configs
./html/plugins/Weathermap/output

**Comment by crazy-max** [comment]
`html/plugins/Weathermap/configs` should not exist in the image. We already create a symlink to `/data/weathermap` with the right permissions: https://github.com/librenms/docker/blob/c95e0d57b681bc3d9b59920ea0681e35c2551b7d/rootfs/etc/cont-init.d/03-config.sh#L87 Check `/data/weathermap` has the right perms.

`html/plugins/Weathermap/output` perms are already fixed at runtime: https://github.com/librenms/docker/blob/c95e0d57b681bc3d9b59920ea0681e35c2551b7d/rootfs/etc/cont-init.d/02-fix-perms.sh#L12

**Comment by murrant** [comment]
I see.  So the issue is if the /data/weathermap directory has bad permissions?

**Comment by murrant** [comment]
Unfortunately, people don't read notes.  How about just adding chown on /data/weathermap?

**Comment by crazy-max** [comment]
> Unfortunately, people don't read notes. How about just adding chown on /data/weathermap?

Don't think a chown on `/data/weathermap` folder will be enough as files under it will still have wrong perms.

**Comment by murrant** [comment]
How about this @crazy-max ?

**Comment by murrant** [comment]
I reviewed the files a little more and /data/weathermap already has a chown (non-recursive).  I can't see why this PR is needed.

**Pull request closed without merging**